pkg/pkiaas/api/template_controller.go

```diff
@@ -41,7 +41,8 @@ func CreateTemplateHandler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, httpErr.JSON(), httpErr.HTTPResponse)
 		return
 	}
-	var newTemplate types.Template
+
+	newTemplate := &types.Template{StoreCertificate: true}
```

Comment:
Here it is again...

Reply:
It defaults to true. If false is provided it overrides the default.